tigerpath/majors_and_certificates/majors/EEB_2018.json

```
@@ -55,37 +56,63 @@
           ]
         },
         {
-          "name": "Math",
+          "name": "Chemistry",
           "max_counted": 1,
-          "min_needed": 1,
-          "explanation": "Mastery of calculus to the level of MAT 103 or above, or advanced placement (an AP score of 5 on the AB test or an AP score of 4 on the BC test);",
-          "course_list": [
-            "MAT 103",
-            "MAT 104",
-            "MAT 201",
-            "MAT 202",
-            "MAT 203",
-            "MAT 204"
+          "min_needed": 2,
+          "explanation": "Two terms of introductory chemistry (or equivalent, or an AP score of 5).",
+          "req_list": [
+            {
+              "name": null,
+              "max_counted": null,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "CHM 202"
+              ]
+            },
+            {
+              "name": null,
+              "max_counted": 1,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "CHM 201",
+                "CHM 207"
+              ]
+            },
+            {
+              "name": null,
+              "max_counted": 2,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "CHM 215"
+              ]
+            }

```

Even though `max_counted` is 2, you still need to add the course twice to get there.

A cool trick is to duplicate the hidden subrequirement and then set its parent's `double_counting_allowed` to true. (In fact why not set it for the full Prerequisites requirement, as they don't overlap with departmentals anyway?)
Doing so lets the course count as double. The only downside is that it also appears doubled in the requirements tree, but that's probably not an issue unless you think it would confuse people.
```suggestion
            {
              "name": null,
              "max_counted": 1,
              "min_needed": null,
              "explanation": null,
              "course_list": [
                "CHM 215"
              ]
            },
            {
              "name": null,
              "max_counted": 1,
              "min_needed": null,
              "explanation": null,
              "course_list": [
                "CHM 215"
              ]
            },
```

Up to you if you think this is inelegant, maybe there's another way.